components/pkg-export-kubernetes/src/manifest.rs

```diff
@@ -28,20 +28,39 @@ use topology::Topology;
 use manifestjson::ManifestJson;
 use bind;
 
+/// Represents a Kubernetes manifest.
 #[derive(Debug, Clone)]
 pub struct Manifest {
+    /// The service name in the Kubernetes cluster.
     pub metadata_name: String,
+    /// The service name in the Habitat ring.
```

Comment:
I think the difference between these two name fields should be explained better. I don't understand it for example.

Reply:
Sure, that way maybe I'll get it in my head too. :)